src/nvim/fileio.c

```diff
@@ -5461,21 +5472,23 @@ find_end_event (
   return pat;
 }
 
-/*
- * Return TRUE if "event" is included in 'eventignore'.
- */
-static int event_ignored(event_T event)
+/// Return true if "event" is included in 'eventignore'.
+///
+/// @param event The event to check
+static bool event_ignored(event_T event)
```

Comment:
FUNC_ATTR_PURE, possibly FUNC_ATTR_WARN_UNUSED_RESULT (be consistent with previous decision).